tests/src/e2e/ledger_tests.rs

```diff
@@ -247,48 +293,59 @@ fn invalid_transactions() -> Result<()> {
     // 1. Run the ledger node
     let mut ledger =
         run_as!(test, Who::Validator(0), Bin::Node, &["ledger"], Some(20))?;
+    let keypair = wallet::defaults::daewon_keypair();
+    let daewon = keypair.to_string();
     ledger.exp_string("Anoma ledger node started")?;
-    ledger.exp_string("Started node")?;
+    if !cfg!(feature = "ABCI") {
+        ledger.exp_string("started node")?;
+    } else {
+        ledger.exp_string("Started node")?;
+    }
     // Wait to commit a block
-    ledger.exp_regex(r"Committed block hash.*, height: [0-9]+")?;
+    // ledger.exp_regex(r"Committed block hash.*, height: [0-9]+")?;
```

Comment:
Does this still work?
```suggestion
    ledger.exp_regex(r"Committed block hash.*, height: [0-9]+")?;
```